apps/fz_http/lib/fz_http.ex

```diff
@@ -1,9 +1,21 @@
 defmodule FzHttp do
-  @moduledoc """
-  FzHttp keeps the contexts that define your domain
-  and business logic.
+  def schema do
+    quote do
+      use Ecto.Schema
 
-  Contexts are also responsible for managing your data, regardless
-  if it comes from the database, an external API or others.
+      @primary_key {:id, :binary_id, autogenerate: true}
```

Comment:
Thoughts on PG-generated `uuid`s vs Ecto generating them? I could see pros and cons to each.

Reply:
Both are generating v4 UUIDs, I don't know what source of entropy they are using but for us, they will be practically the same.

Reply:
If any other system creates those, PG can still autogenerate ids (because the PK autogenerate option is set automatically on the DB level). Plus, if other apps generate valid UUIDv4 it doesn't matter where they are generated - the standard is the same.